src/components/Pagination/Link.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { createElement as E, Component } from 'react'
+import T from 'prop-types'
+import { classNames } from '../../utils/'
+
+class PaginationLink extends Component {
+  handleClick = (e) => {
+    this.props.callback(e, this)
+  }
+
+  render() {
+    const {
+      to,
+      component,
+      node,
+      className,
+      isActive,
+      activeClassName = 'is-active',
+      callback,
+      children,
+      togglePreviousNext,
+      ...rest
+    } = this.props
+
+    const onClickProps = callback
+      ? {
+        onClick: this.handleClick
+      } : {}
+
+    const ComponentOverride = component
+
+    if (ComponentOverride) {
+      return (
+        <ComponentOverride
+          to={to}
+          className={classNames('c-pagination__link', className)}
+          activeClassName={activeClassName}
+          togglePreviousNext={togglePreviousNext}
```

Comment:
prop plus corresponding tests will need updating to mach the prop change.

Reply:
done